src/main/java/com/labsynch/labseer/service/ExperimentServiceImpl.java

```diff
@@ -1118,6 +1118,45 @@ public Collection<Experiment> findExperimentsByGenericMetaDataSearch(String quer
 
 	}
 
+	public Collection<Experiment> findExperimentsByProtocolCodeName(String codeName, List<String> projects)
+			throws TooManyResultsException, NoResultException {
```

Comment:
Just curious - how would TooManyResultsException get thrown here?

Reply:
Good point! It doesn't, I accidentally throw a RunTimException instead.

Reply:
Fixed